tests/TestCase/Command/StatusTest.php

```diff
@@ -133,9 +130,9 @@ public function testExecuteJson()
         $commandTester->execute(['command' => $this->command->getName()] + $params);
         $display = $this->getDisplayFromOutput();
 
-        $expected = '{"status":"down","id":"20150704160200","name":"CreateNumbersTable"},'
-            . '{"status":"down","id":"20150724233100","name":"UpdateNumbersTable"},'
```

Comment:
I much preferred the first as it is better to read. Does the sniffer actually want this to change?

Reply:
Yes. `ERROR | Expected 1 space before ., but 12 found`

Reply:
Well, the sniffer is wrong here IMO and should be fixed, not the code.

Reply:
I'm not sure. To me the sniffer suggestion looks fine. Actually I'm fine with both.

Reply:
I opened a ticket https://github.com/cakephp/cakephp-codesniffer/issues/198 for now.

Reply:
@dereuromark It's ours it seems:

` 134 | ERROR | Expected 1 space before ., but 12 found (CakePHP.Strings.ConcatenationSpacing.TooManyBefore)
`